simulation-runner/src/main/scala/com/puppetlabs/gatling/node_simulations/FOSS375CatalogZero.scala

```diff
@@ -0,0 +1,277 @@
+package com.puppetlabs.gatling.node_simulations
+
+import scala.concurrent.duration._
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import io.gatling.jdbc.Predef._
+
+import com.puppetlabs.gatling.runner.SimulationWithScenario
+import org.joda.time.LocalDateTime
+import org.joda.time.format.ISODateTimeFormat
+
+class FOSS375CatalogZero extends SimulationWithScenario {
+
+	val httpProtocol = http
```

Comment:
In some of the other sims, I've seen `httpProtocol`, `uri1`, and the `setup` call at the end of this commented out.  Do we want that here too?